### DISCUSSION TITLE
Imad Dabbura - Why Having the Last Layer in a Neural Network as Raw Logits is Better Than Softmax

### DISCUSSION THREAD

**Discussion by giscus** [ImadDabbura/imaddabbura.github.io#11] [General]
# Imad Dabbura - Why Having the Last Layer in a Neural Network as Raw Logits is Better Than Softmax

https://imaddabbura.github.io/posts/dl/why-not-softmax.html
